.github/.scripts/test_launch_app.py

```diff
@@ -0,0 +1,113 @@
+import json
```

Comment:
this file location feels a bit off - it feels more like an integration test in flavor. Is it possible to move this under a non .github directory and make this work? Because I wouldn't know to look here for a test